functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/MisspelledStrategiesFunctionalTest.java

```diff
@@ -24,7 +25,7 @@
     public TestBed testBed = new TestBed(GIT_CLONE);
 
     @Test
-    public void should_fail_and_log_suggestion_for_misspelled_strategies() throws Exception {
+    public void should_fail_and_log_suggestion_for_misspelled_strategies_when_autocorrect_disabled() throws Exception {
```

Comment:
I don't see anything in this test which is throwing exception

Reply:
There is no exception since it is a ftest, the error is found in the Maven log.